src/insns/atomic_exceptions.adoc

```diff
@@ -26,9 +26,9 @@ reported in the CAUSE field of <<mtval>> or <<stval>>:
 | CAUSE                 | Reason
 | Tag violation         | Authority capability tag set to 0
 | Seal violation        | Authority capability is sealed
-| Permission violation  | Authority capability does not grant <<r_perm>> or <<w_perm>>
+| Permission violation  | Authority capability does not grant <<r_perm>> or <<w_perm>>, or the AP field could not have been produced by <<ACPERM>>
 | Invalid address violation  | The effective address is invalid according to xref:section_invalid_addr_conv[xrefstyle=short]
-| Length violation      | At least one byte accessed is outside the authority capability bounds
+| Length violation      | At least one byte accessed is outside the authority capability bounds, which includes the case where the capability is <<section_cap_malformed,malformed>> as the bounds decode as zero
```

Comment:
I'd cut "as the bounds decode as zero": I think it just confuses things, though in this case it is accurate: no accesses are in bounds for [0, 0)

Reply:
done